### PR TITLE
Add config flag to optionally pass explicit config path to cli

### PR DESCRIPTION
**Pull request by mahirshah**
Add --config flag to checkup cli that allows users to optionally pass in
an explcit path to the config file to be loaded.

### CODE REVIEW

File: packages/cli/README.md [hunk 1]
```diff
@@ -44,17 +44,22 @@ checkup is designed to be completely configurable via a configuration object.
...
-- checkup.yaml / .checkup.yml
-- checkup.js
+- .checkup.json
+- .checkup.yaml / .checkup.yml
+- .checkup.js
+
+You can also specify an explicit path to a configuration via the command line, which will override any configurations found in any `.checkuprc.*` files
+
+```sh-session
+$ checkup --config /some/path/to/my/config/.eslintrc
```
Comment (scalvert) [inline packages/cli/README.md]:
```suggestion
$ checkup --config /some/path/to/my/config/.checkuprc
```

File: packages/cli/__tests__/index-test.ts [hunk 2]
```diff
@@ -53,13 +53,37 @@ describe('@checkup/cli', () => {
           },
         })
         .build();
+      const anotherPlugin = new Plugin.PluginBuilder('another-plugin-mock')
+        .addTask('MockTask3', {
```
Comment (scalvert) [inline packages/cli/__tests__/index-test.ts]: I think you're going to have a conflict here with my changes that were just merged. Easy to update.

File: packages/core/__tests__/configuration/get-filepath-loader-test.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  afterEach(() => {
+    project.dispose();
+  });
+
+  it('should throw if a config file is not found at the given filepath', async () => {
+    project.writeSync();
+    await expect(
+      getFilepathLoader(path.join(project.baseDir, '.checkuprc'))()
+    ).rejects.toThrowErrorMatchingInlineSnapshot(
+      `"ENOENT: no such file or directory, open '${path.join(project.baseDir, '.checkuprc')}'"`
```
Comment (scalvert) [inline packages/core/__tests__/configuration/get-filepath-loader-test.ts]: I feel like we should catch this error and rethrow with a better message. Thoughts?

File: packages/cli/README.md (hunk 1 above)
Comment (mahirshah) [reply packages/cli/README.md]: good eye, fixed!

**Pull request merged**